src/workflow/workflow.c

```diff
@@ -73,11 +74,13 @@ static void _select_orientation_done(bool upside_down)
     component_t* show_logo = show_logo_create(workflow_start, 200);
     ui_screen_stack_switch(show_logo);
 }
+#endif
 
 void workflow_start_orientation_screen(void)
 {
 #if PLATFORM_BITBOXBASE == 1
-    _select_orientation_done(false);
+    component_t* show_logo = show_logo_create(workflow_start, 200);
+    ui_screen_stack_switch(show_logo);
```

Comment:
Direct call to `_select_orientation_done(false)` instead? So we don't have to track future changes in the latter.

Reply:
Yeah, it's either that and doing ifdefs around "upside_down" or copying these two lines. I did the second. I think the whole "orientation screen" doesn't make sense from the BBB point of view.